index.bs

```diff
@@ -13616,6 +13623,189 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 </div>
 
+## The autofill Module ## {#module-autofill}
+
+The <dfn export for=modules>autofill module</dfn> contains functionality for triggering autofill.
+
+### Definition ### {#module-autofill-definition}
+
+{^remote end definition^}
+
+<pre class="cddl" data-cddl-module="remote-cddl">
+
+AutofillCommand = (
+  autofill.Trigger
+)
+</pre>
+
+### Commands ### {#module-autofill-commands}
+
+#### The autofill.trigger Command ####  {#command-autofill-trigger}
+
+The <dfn export for=commands>autofill.trigger</dfn> command triggers autofill
+on a particular form field using the provided fields.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      autofill.Trigger = (
+        method: "autofill.trigger",
+        params: autofill.TriggerParameters
+      )
+
+      autofill.TriggerParameters = {
+        context: browsingContext.BrowsingContext,
+        element: script.SharedReference,
```

Comment:
Autofill (consisting of form/field classification, data selection, and filling) all happens for the entire form, not one element. Many of these field tokens are commonly split across multiple form fields, and sometimes across multiple forms. (Also, <input> with no <form>.) So I'm not sure what to say about this parameter except it's unnecessary IMO.